src/workflows/steps/index.js

```diff
@@ -1373,7 +1373,10 @@ ${chalk.green(log)}`);
 		state.step = "promptKeepBranchOrCreateNew";
 		const { log, branch } = state;
 
-		if (!log.length) {
+		if (
+			!log.length ||
+			(branch === "develop" || branch === "master" || branch === "main")
```

Comment:
Since these are all or'ed logic the grouping shouldn't be needed...

```
if ( !log.length || branch === "develop" || branch === "master" || branch === "main" ) {}
```

Reply:
Or you could define an array of excluded branches and check that...

```
const LOCKED_BRANCHES = [ "develop", "master", "main" ];
if ( !log.length || LOCKED_BRANCHES.includes( branch ) ) {}
```